circle.yml

```diff
@@ -0,0 +1,18 @@
+dependencies: 
+  cache_directories: 
+    - ~/.cache/pip
+    - ~/wheelhouse
+  post: 
+    - .circle/services.sh
+    - ".circle/compose.sh run wheezy /bin/true"
+    - ".circle/compose.sh run wheezy"
+  pre: 
+    - .circle/cache-permissions.sh
+
+machine: 
+  services: 
+    - docker
+
+test:
+  override:
+    - ".circle/compose.sh run wheezy bash -c 'cp /root/Gemfile* ./ && bundle exec rspec'"
```

Comment:
It could be more useful to have something like: 
`docker-compose -f compose.yml -f compose.circle.yml run wheezy` without wrapper wrapper script.

Is it possible to move `cp /root/Gemfile* ./ && bundle exec rspec` logic into `compose.circle.yml` ?